src/AppBundle/Resources/views/templates/artist-detail.html.twig

```diff
@@ -7,8 +7,61 @@
         {# views/master.html.twig and views/templates/homepage.html.twig. #}
         {% include('AppBundle:includes:headerimage.html.twig') %}
 
-        {% include ('AppBundle:renderer:block-renderer.html.twig') with {
-            'blocks' : content.blocks
-        } %}
+        <div class="grid content">
+            <div class="grid__item desk--eight-twelfths">
+                {% include ('AppBundle:renderer:block-renderer.html.twig') with {
+                    'blocks' : content.blocks
+                } %}
+            </div>
+
+            <div class="grid__item desk--four-twelfths">
+                <div class="facts">
+                    <div class="facts-header">Facts</div>
+                    <div class="facts-content">
+                        <p>
+                            {% if content.genre|default() %}
```

Comment:
Nevermind, just saw that it has been done everywhere like that, probably not worth changing it.